.github/workflows/ci_checks.yaml

```diff
@@ -99,21 +175,6 @@ jobs:
       - name: Test in firefox
         run: wasm-pack test --headless --firefox ./bindings/wallet-js
 
-  test-key-gen-lib:
```

Comment:
Why was this removed?

Reply:
This js library is no longer needed by Daedalus, if it ever used at all; I think the developers figured out how to generate ed25519 keys without a weird Rust library to help.

Reply:
should we also remove the actual code?

Reply:
Nobody's asking for a new release of the keygen package, so I think it's time.